test/sandbox-test.js

```diff
@@ -243,6 +243,28 @@ describe("sinonSandbox", function () {
         }
     });
 
+    describe(".restore", function () {
+        it("throws when passed arguments", function () {
+            this.sandbox = sinonSandbox.create();
+
+            var failed = false;
```

Comment:
I was surprised to see that referree doesn't have an `assert.throws` function. I found this pattern elsewhere in your test suite.